frappe/commands/utils.py

```diff
@@ -772,12 +772,11 @@ def run_tests(
 			click.secho(f"bench --site {site} set-config allow_tests true", fg="green")
 			return
 
-		frappe.init(site=site)
```

Comment:
Breaks `bench run-tests` since we need to access `frappe.flags` right below here